galaxy/importer/tests/test_collection_info.py

```diff
@@ -86,18 +86,18 @@ def galaxy_col_info(base_col_info):
 ]
 
 valid_licenses = [
-    'MIT',
-    'Apache-2.0',
-    'BSD-3-Clause',
-    'CC-BY-4.0'
+    ['MIT'],
+    ['Apache-2.0'],
+    ['BSD-3-Clause'],
+    ['CC-BY-4.0', 'MIT']
```

Comment:
+1 For included an intentional dupe in the test data list.  Good way to catch potential assumptions about data ordering and uniqueness.

For this particular case, in theory the valid_licenses is supposed to be an arbitrarily ordered (but usually sorted) list derived from the keys of a dict. But it is easy for the app code to have bugs because of assumptions about the data, so good thing to test for.